Rocket.Chat/Extensions/Models/SubscriptionExtensions.swift

```diff
@@ -25,6 +25,6 @@ extension Results where Element == Subscription {
     }
 
     func filterBy(name: String) -> Results<Subscription> {
-        return self.filter("name CONTAINS %@", name)
+        return self.filter("name CONTAINS[c] %@", name)
```

Comment:
Do we have tests on this methods? I think would be something nice to have here if we don't. 👍